### DISCUSSION TITLE
Connect github copilot with vscode

### DISCUSSION THREAD

**Discussion by nt-hiep** [github/copilot-docs#68] [Polls]
I try to authorize with github but it does not work

**Comment by allan1366** [comment]
来信收到，谢谢！张韶华

**Comment by nguyen-tuan-hiep** [comment]
you need to have access to github copilot, try to verify with github education for example

**Reply by nt-hiep** [reply]
thank you

**Comment by brayanjeshua** [comment]
Just install the extension and sync with your github account
